apps/wallet-mobile/src/features/Staking/Governance/useCases/ChangeVote/ChangeVoteScreen.tsx

```diff
@@ -30,7 +30,9 @@ export const ChangeVoteScreen = () => {
   const action = stakingStatus ? mapStakingKeyStateToGovernanceAction(stakingStatus) : null
   const {openModal} = useModal()
   const {manager} = useGovernance()
-  const [pendingVote, setPendingVote] = React.useState<GovernanceVote['kind'] | null>(null)
+  const [pendingVote, setPendingVote] = React.useState<
+    'abstain' | 'no-confidence' | 'delegate-to-yoroi' | 'delegate-not-yoroi' | null
```

Comment:
local UI types, so I considered not creating a global type for this

Reply:
makes sense.